packages/table/src/table.tsx

```diff
@@ -670,6 +682,22 @@ export class Table extends AbstractComponent<ITableProps, ITableState> {
         });
     }
 
+    // Quadrant refs
+    // =============
+
+    private generateQuadrantRefHandlers(quadrantType: QuadrantType): IQuadrantRefHandlers {
```

Comment:
This is pretty funky. Can we do this either iteratively over the four items, or... something else? We're sortof in the wrong place between re-useable code and not, here.